std/node/_fs/_fs_dir.ts

```diff
@@ -17,6 +17,7 @@ export default class Dir {
   }
 
   read(callback?: Function): Promise<Dirent | null> {
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
Remove this ignore and async executor below

Reply:
done 👍